app/src/main/java/org/mozilla/rocket/content/travel/data/TravelRemoteDataSource.kt

```diff
@@ -70,10 +70,14 @@ class TravelRemoteDataSource : TravelDataSource {
         TODO("not implemented")
     }
 
-    override suspend fun getCityHotels(cityId: String): Result<BcHotelApiEntity> = withContext(Dispatchers.IO) {
+    override suspend fun getCityHotels(cityId: String, offset: Int): Result<BcHotelApiEntity> = withContext(Dispatchers.IO) {
```

Comment:
I think we could encapsulate the 100-base offset logic in the data source layer by exporting the `page` instead `offset` parameter.